keyboards/dz60/keymaps/heather/keymap.c

```diff
@@ -0,0 +1,76 @@
+#include QMK_KEYBOARD_H
+
+//Layers
+#define _DEFAULT 0
+#define _FN 1
+#define _MEDIA 2
+
+//Filler for KC_TRNS/KC_NO just so the key maps are a bit more readable
+#define ______ KC_TRNS
+#define XXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```